### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by TanThiHa**
ready

### CODE REVIEW

**Comment by TanThiHa** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+/* universal */
+
+>>>>>>> finish chapter 6
```
Comment (phamvanbk11) [inline app/assets/stylesheets/custom.scss]: fix conflict có tâm nhỉ

**Comment by phamvanbk11** [comment]
commented

**Comment by TanThiHa** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**